### ISSUE TITLE
ISO Packaged for LiveUSB, but Install Linux, does not work

### ISSUE THREAD

**Issue by PetRose** [unetbootin/unetbootin#36]
Hi there
I have made a LiveUSB (using the Windows version 6.1.3), package of Linux Mint 13, (32bit, Maya Xfce), and when it boots the target system two laptops (one with Windows Filesystem, another with Linux Partitions on their HDD), nothing happens when i press the Desktop shortcut to 'Install Linux Mint' ?
On thhe Linux Partition Laptop, the Live version says right away:
    Not enough space on /var/log/installation
The other Laptop, doesn't give any clue, in fact the Desktop shortcut, seems not to be the same.
Where is the problem, you think, in the:
1. unetbootin software ?
2. in the ISO packaged with unetbootin ?
3. both ?
Thanks, in advance

**Comment by PetRose** [comment]
I believe this issue can be withdrawn. 
After I did re-use my FlashDrive, saying Yes to overwrite files, and then experienced the reported issue. I thought I would give it another try, by reformatting flashdrive (FAT-32, 2K Blocksize) and then created my decided Linux Mint 32 bit ISO with Unetbootin.
After successful reboot on the target laptop, this time I did manage to get the Install shortcut to respond, and allow me to make an Install. So for now, I'm happy.
